adafruit_hid/keycode.py

```diff
@@ -33,7 +33,7 @@ class Keycode:
     """USB HID Keycode constants.
 
     This list is modeled after the names for USB keycodes defined in
-    https://www.usb.org/sites/default/files/documents/hut1_12v2.pdf#page=53.
+    https://www.usb.org/sites/default/files/hut1_21_0.pdf#page=118.
```

Comment:
This is the newer version of the other document, and the page. It's the Keyboard/Keypad Page (0x07). 118 is the Consumer Control codes page.